release-tools/src/main/java/org/springframework/data/release/build/MavenBuildSystem.java

```diff
@@ -277,18 +271,72 @@ public ModuleIteration prepareVersion(ModuleIteration module, Phase phase) {
 		return module;
 	}
 
-	/*
-	 * (non-Javadoc)
-	 * @see org.springframework.data.release.build.BuildSystem#deploy(org.springframework.data.release.model.ModuleIteration)
+	/**
+	 * Perform a {@literal nexus-staging:rc-open} and extract the stagingProfileId from the results.
 	 */
 	@Override
+	public void open() {
+
+		try {
+			CommandLine arguments = CommandLine.of(Goal.goal("nexus-staging:rc-open"), //
+					profile("central"), //
+					of("-s " + properties.getSettingsXml()), //
```

Comment:
It would be good to have the settings XML as pre-defined argument for reuse in the other goals.